src/main/java/com/ericsson/ei/frontend/EIRequestsController.java

```diff
@@ -165,6 +165,10 @@
             LOG.debug("EI Http response status code: " + statusCode
                     + "\nEI Received response body:\n" + responseBody
                     + "\nForwarding response back to EI Frontend WebUI.");
+        } catch (java.lang.NullPointerException e) {
+            // This is most likely an error during test so we make it silent.
+            statusCode = HttpStatus.INTERNAL_SERVER_ERROR.value();
+            LOG.warn("Forward Request Warning: " + e);
```

Comment:
Since both these catch functions is very similar, maybe both these catches can be changed to one catch: "catch (Exception e) { ......... }"